README.md

```diff
@@ -1062,6 +1062,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [formspark.io](https://formspark.io/) -  Form to Email service, free plan allows unlimited forms, 250 submissions per month, support by Customer assistance team.
   * [Formspree.io](https://formspree.io/) — Send email using an HTTP POST request. The free tier limits to 50 submissions per form per month.
   * [Formsubmit.co](https://formsubmit.co/) — Easy form endpoints for your HTML forms. Free Forever. No registration is required.
+  * [Formlick.com](https://formlick.com) - Typeform alternative with lifetime deal. Users can create unlimited forms and recieve unlimited submissions.
```

Comment:
Free is only 1 form, please put the limits of the free plan here not the paid plan

Reply:
fixed